walletlibrary/src/main/java/com/microsoft/walletlibrary/util/Exceptions.kt

```diff
@@ -59,6 +59,8 @@ class VerifiedIdRequirementNotFulfilledException(message: String = "", cause: Th
 
 class VerifiedIdTypeIsNotRequestedTypeException(message: String = "", cause: Throwable? = null, retryable: Boolean = false): RequirementValidationException(message, cause, retryable)
 
+class VerifiedIdIssuerIsNotRequestedException(message: String = "", cause: Throwable? = null, retryable: Boolean = false): RequirementValidationException(message, cause, retryable)
```

Comment:
naming, what does this mean? Does every VerifiedId Issuer have to be requested?

Reply:
I started off as supporting constraints for Accepted Issuer so added this exception. Now this supports claims as well. I will rename it to reflect that.